docs/getting-started-guides/windows/README.md

```diff
@@ -0,0 +1,159 @@
+# Running Windows Server Containers using Kubernetes
+Windows Server Containers are supported on Kubernetes as an alpha feature. Kubernetes control plane (API Server, Scheduler, Controller Manager etc) will continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server.
+
+## Prerequisites
+With the alpha release, Windows Server Containers for Kubernetes is supported using the following:
+
+1. Kubernetes control plane running on existing Linux infrastructure (version 1.5 or later)
+2. Kubenet network plugin setup on the Linux nodes
+3. Windows Server 2016 (RTM version 10.0.14393 or later)
+4. Docker Version 1.12.2-cs2-ws-beta or later
+
+## Networking
+Network is achieved using L3 routing. Because third-party networking plugins (e.g. flannel, calico, etc) don’t natively work on Windows Server, existing technology that is built into the Windows and Linux operating systems is relied on. In this L3 networking approach, a /16 subnet is chosen for the cluster nodes, and a /24 subnet is assigned to each worker node. All pods on a given worker node will be connected to the /24 subnet. This allows pods on the same node to communicate with each other. In order to enable networking between pods running on different nodes, routing features that are built into Windows Server 2016 and Linux are used.
+
+### Linux
+The above networking approach is already supported on Linux using a bridge interface, which essentially creates a private network local to the node. Similar to the Windows side, routes to all other pod CIDRs must be created in order to send packets via the “public” NIC.
+
+### Windows
+Each Window Server node should have the following configuration:
+
+1. Two NICs (virtual networking adapters) are required on each Windows Server node - The two Windows container networking modes of interest (transparent and L2 bridge) use an external Hyper-V virtual switch. This means that one of the NICs is entirely allocated to the bridge, creating the need for the second NIC.
+2. Transparent container network created - This is a manual configuration step and is shown in **_Route Setup_** section below
+3. RRAS (Routing) Windows feature enabled - Allows routing between NICs on the box, and also “captures” packets that have the destination IP of a POD running on the node. To enable, open “Server Manager”. Click on “Roles”, “Add Roles”. Click “Next”. Select “Network Policy and Access Services”. Click on “Routing and Remote Access Service” and the underlying checkboxes
+4. Routes defined pointing to the other pod CIDRs via the “public” NIC - These routes are added to the built-in routing table as shown in **_Route Setup_** section below
+
+The following diagram illustrates the Windows Server networking setup for Kubernetes Setup
+![Windows Setup](windows-setup.png)
+
+## Setup
```

Comment:
The reason I have 3 successive nested headings is because Setup have 3 types Host, Component and Route. Host and Route instructions are different for Windows and Linux Hosts and Components setup have instructions for building `kubelet` and `kube-proxy` for Windows. So as each Setup type have 2 more child elements I thought it is easier to follow if it is nested headings.